java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -2996,4 +2997,24 @@ public void pfmerge_returns_success() {
         assertEquals(testResponse, response);
         assertEquals(OK, response.get());
     }
+
+    @SneakyThrows
+    @Test
+    public void objectEncoding_returns_success() {
+        // setup
+        String key = "testKey";
+        String encoding = "testEncoding";
+        CompletableFuture<String> testResponse = new CompletableFuture<>();
+        testResponse.complete(encoding);
+        when(commandManager.<String>submitNewCommand(eq(ObjectEncoding), eq(new String[] {key}), any()))
```

Comment:
Nit: keep formatting the same
```suggestion

        // match on protobuf request
        when(commandManager.<String>submitNewCommand(eq(ObjectEncoding), eq(new String[] {key}), any()))
```